nautobot/extras/forms.py

```diff
@@ -1047,6 +1054,24 @@ def clean(self):
                     }
                 )
 
+            if cleaned_data.get("_schedule_type") == JobExecutionType.TYPE_CUSTOM:
+                if not cleaned_data.get("_recurrence_custom_time"):
+                    raise ValidationError({"_recurrence_custom_time": "Please provide a crontab schedule."})
+
+                try:
+                    minute, hour, day_of_month, month_of_year, day_of_week = cleaned_data.get(
+                        "_recurrence_custom_time"
+                    ).split(" ")
+                    schedules.crontab(
+                        minute=minute,
+                        hour=hour,
+                        day_of_month=day_of_month,
+                        month_of_year=month_of_year,
+                        day_of_week=day_of_week,
+                    )
+                except Exception as e:
+                    raise ValidationError({"_recurrence_custom_time": e})
```

Comment:
This appears to be duplicating the same code as is found in `ScheduledJob.get_crontab()`. Can we not just use that same method? There's nothing on that method that requires an instance, so it could be converted to a `class method` and then called here directly from the model class object a la `ScheduledJob.get_crontab()`.